resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/providers/sse/SseEventInputImpl.java

```diff
@@ -118,11 +120,11 @@ public InboundSseEvent read() throws IllegalStateException
             {
                temSave.write(b);
                b = readLine(entityStream, ':', temSave);
-               String fieldName = temSave.toString(charset.toString());
+               String fieldName = temSave.toString(StandardCharsets.UTF_8.name());
                temSave.reset();
                if (b == ':')
                {
-                  //space after the colon is ignored
+                  //spec says there is space after colon
                   do
```

Comment:
Actually, it says, "If value starts with a U+0020 SPACE character, remove it from value."  That is, it seems to be saying to remove only a single space.

Reply:
@ronsigal  I looked at this in spec. I think more blank spaces after the filed name is useless.  The field value should be started with non blank spaces content. I can't think of an event case which the field value should be started with blank space.

Reply:
@jimma I don't disagree with you, I'm just repeating from the spec.